### PR TITLE
Add thelio-major-b2 service manual

### PR DESCRIPTION
**Pull request by jackpot51**
Redo of #12 using git lfs

### CODE REVIEW

**Comment by storyaddict** [comment]
Most of these things have now been resolved; some will have to wait until a v2.0 if we're to get this out and get started the Massive finished.

-----

Removing the GPU
Should probably start with 
"In this guide, you will learn how to remove the graphics card on your Thelio. These steps must be taken in order to replace the GPU. You can find video instructions on this process here.**


Link for the video for GPU removal: (https://youtu.be/3bUcp0S1HrE)"

keep the tools section for the GPU the same as all the others. You can include the bolded text disclaimer in that area and with any images that describe that. **[RESOLVED - Disclaimer moved]**

To remove your graphics card, twist off the 3 thumbscrews from the PCI bracket.
Needs an image **[PUSH to manual v.20 - not currently have a photo of this]**

Remove the GPU brace. Unscrew the bottom two screws, and then the top two screws, to remove the brace. Both the PCI bracket and the GPU brace keep your graphics card secure and in place (Fig 23).

Remove the GPU from the PCIe slot by pressing on the release clip on the right. This is where your chosen (non-metal) poking tool may come in handy, to reach the clip on the board without trying to squeeze your hand between components.

Needs an image **[Partially RESOLVED - have highlights showing the release clip, but no pictures of using a tool to release].**

Disconnect the power cables from the GPU (if applicable), and tie back any cables you are not planning on using with the provided velcro straps.
Needs an image **[Push to v2.0 - videos are enough for now]**

Fig: 30 Showing variations of pin connections on GPU cards - 8-pin (top), 12-pin (middle), 16-pin (bottom).

Consolidate into one image and remove all the excess **[RESOLVED]**

Once the GPU is connected and the power cables secured,
Needs Image showing cables properly connected to power supply and GPU **[RESOLVED]**

Turn the power supply switch at the rear of the machine to the “on” position.

Gently but firmly separate the Cooling Fans from the Heat Sink (fig. 33). -make sure to describe underneath image **[PUSH to v2.0]**

Next, disconnect the power cables from the Cooling Fans.
Needs image **[RESOLVED]**

Thermal Paste:
'NOTE: DO NOT use your Thelio without thermal paste. Disclaimer should be at the beginning of the section, needs more images and descriptions under images' **[RESOLVED]**

**Comment by Socialhappiness** [comment]
Love the changes, you guys did an awesome job!

File: src/models/addw1/README.md [hunk 1]
```diff
@@ -1,26 +0,0 @@
-# Adder WS (addw1)
```
Comment (jackpot51) [inline src/models/addw1/README.md:1]: Was deleting these files intentional?

File: src/models/lemp9/README.md [hunk 2]
```diff
@@ -1,50 +0,0 @@
-# Lemur Pro (lemp9)
```
Comment (jackpot51) [inline src/models/lemp9/README.md:1]: Was deleting this file intentional?

File: src/models/addw1/README.md (hunk 1 above)
Comment (ahoneybun) [reply src/models/addw1/README.md:1]: No.

File: src/models/lemp9/README.md (hunk 2 above)
Comment (ahoneybun) [reply src/models/lemp9/README.md:1]: No.

File: src/models/addw1/README.md (hunk 1 above)
Comment (storyaddict) [reply src/models/addw1/README.md:1]: yes, they shouldn't have been this branch. They were causing conflicts with the master branch.

File: src/models/lemp9/README.md (hunk 2 above)
Comment (storyaddict) [reply src/models/lemp9/README.md:1]: yes, they shouldn't have been in this branch. They were causing conflicts with the master branch.

**Comment by jackpot51** [comment]
The conflicts with this need to be resolved, and it needs to be squashed into one commit.

**Pull request closed without merging**